syntax/std_seq.go

```diff
@@ -64,5 +64,192 @@ func stdSeq() rel.Attr {
 	return rel.NewTupleAttr("seq",
 		rel.NewNativeFunctionAttr("concat", stdSeqConcat),
 		rel.NewNativeFunctionAttr("repeat", stdSeqRepeat),
+		createNestedFuncAttr("contains", 2, func(args ...rel.Value) rel.Value { //nolint:dupl
+			return includingProcess(func(args ...rel.Value) rel.Value {
+				sub, subject := args[0], args[1]
+				return rel.NewBool(strings.Contains(mustAsString(subject), mustAsString(sub)))
+			},
+				func(args ...rel.Value) rel.Value {
+					sub, subject := args[0], args[1]
+					return arrayContains(sub, subject.(rel.Array))
+				},
+				func(args ...rel.Value) rel.Value {
+					sub, subject := args[0], args[1]
+					return rel.NewBool(strings.Contains(asString(subject), asString(sub)))
+				},
+				args...)
+		}),
```

Comment:
`includeProcess` overcomplicates things for no benefit. Write the code inline.

```suggestion
			sub, subject := args[0], args[1]
			switch subject.(type) {
			case rel.String:
				return rel.NewBool(strings.Contains(mustAsString(subject), mustAsString(sub)))
			case rel.Array:
				return arrayContains(sub, subject.(rel.Array))
			case rel.Bytes:
				return rel.NewBool(strings.Contains(asString(subject), asString(sub)))
			case rel.GenericSet:
				set, is := sub.(rel.GenericSet)
				return rel.NewBool(is && !set.IsTrue())
			}
			return rel.NewBool(false)
		}),
```